core/src/main/java/org/pentaho/platform/dataaccess/metadata/model/impl/Column.java

```diff
@@ -26,7 +26,7 @@
 public class Column implements IColumn {
 
   private static final long serialVersionUID = 3751750093446278893L;
-  private String id, name;
+  private String id, name, description;
   private String type;
   private List<String> aggTypes = new ArrayList<String>();
```

Comment:
@srallapa this does not match the original PR: https://github.com/pentaho/data-access/pull/1282/files. Please review

Reply:
Fixed this, by backporting https://hv-eng.atlassian.net/browse/BACKLOG-42329 into the SP ticket https://hv-eng.atlassian.net/browse/SP-6628